test/e2e/adminUI/tests/group003List/uxTestListView.js

```diff
@@ -21,184 +21,145 @@ module.exports = {
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
 
-		browser.adminUIListScreen
-			.click('@createFirstItemButton');
+		browser.adminUIListScreen.clickCreateItemButton();
 
-		browser.adminUIApp
-			.waitForInitialFormScreen();
+		browser.adminUIApp.waitForInitialFormScreen();
 
 		browser.adminUIInitialFormScreen.fillFieldInputs({
 			modelTestConfig: NameModelTestConfig,
 			fields: {
-				'name': {value: 'Name Field Test 1'},
-				'fieldA': {firstName: 'First 1', lastName: 'Last 1'},
+				'name': { value: 'Name Field Test 1' },
+				'fieldA': { firstName: 'First 1', lastName: 'Last 1' },
 			}
 		});
 
 		browser.adminUIInitialFormScreen.save();
 
-		browser.adminUIApp
-			.waitForItemScreen();
+		browser.adminUIApp.waitForItemScreen();
 
-		browser.adminUIApp
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
 
-		browser.adminUIListScreen
-			.expect.element('@paginationCount').text.to.equal('Showing 1 Name');
+		browser.adminUIListScreen.assertPageItemCountTextEquals('Showing 1 Name');
 
-		browser.adminUIListScreen
-			.expect.element('@firstItemNameValue').text.to.equal('Name Field Test 1');
+		browser.adminUIListScreen.assertItemFieldValueEquals([
+			{ row: 1, column: 2, name: 'name', value: 'Name Field Test 1', modelTestConfig: NameModelTestConfig }
+		]);
 	},
 	'List view should allow users to create more new list items': function (browser) {
-		browser.adminUIApp
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
 
-		browser.adminUIListScreen
-			.click('@createMoreItemsButton');
+		browser.adminUIListScreen.clickCreateItemButton();
 
-		browser.adminUIApp
-			.waitForInitialFormScreen();
+		browser.adminUIApp.waitForInitialFormScreen();
 
 		browser.adminUIInitialFormScreen.fillFieldInputs({
 			modelTestConfig: NameModelTestConfig,
 			fields: {
-				'name': {value: 'Name Field Test 2'},
-				'fieldA': {firstName: 'First 2', lastName: 'Last 2'},
+				'name': { value: 'Name Field Test 2' },
+				'fieldA': { firstName: 'First 2', lastName: 'Last 2' },
 			}
 		});
 
+		// TODO: refactor
 		browser.adminUIInitialFormScreen.section.form
 			.click('@createButton');
 
-		browser.adminUIApp
-			.waitForItemScreen();
+		browser.adminUIApp.waitForItemScreen();
 
-		browser.adminUIApp
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
 
-		browser.adminUIListScreen
-			.expect.element('@paginationCount').text.to.equal('Showing 2 Names');
+		browser.adminUIListScreen.assertPageItemCountTextEquals('Showing 2 Names');
```

Comment:
I understand why this is the way it is, but will most users not try `assertPageItemCountTextEquals(2)` first?